examples/3. Four-bar linkage/Manifest.toml

```diff
@@ -462,23 +479,26 @@ uuid = "931e9471-e8fb-5385-a477-07ad12718aca"
 version = "0.7.1"
 
 [[deps.MeshCat]]
-deps = ["Base64", "Cassette", "Colors", "CoordinateTransformations", "DocStringExtensions", "FFMPEG", "GeometryBasics", "HTTP", "LinearAlgebra", "Logging", "MsgPack", "Parameters", "Pkg", "PrecompileTools", "Requires", "Rotations", "Sockets", "StaticArrays", "Tar", "UUIDs"]
-git-tree-sha1 = "90472d170283eb309e61db47b7d346a99409ca75"
+deps = ["Base64", "Colors", "CoordinateTransformations", "DocStringExtensions", "FFMPEG", "GeometryBasics", "HTTP", "LinearAlgebra", "Logging", "MsgPack", "Parameters", "Pkg", "PrecompileTools", "Requires", "Rotations", "Sockets", "StaticArrays", "Tar", "UUIDs"]
+git-tree-sha1 = "c109e00dbcb1221e38187d36eb4579b0dfb5b918"
 uuid = "283c5d60-a78f-5afe-a0af-af636b173e11"
-version = "0.16.2"
+version = "1.0.0"
 
 [[deps.MeshCatMechanisms]]
 deps = ["ColorTypes", "CoordinateTransformations", "GeometryBasics", "InteractBase", "Interpolations", "LoopThrottle", "MechanismGeometries", "MeshCat", "OrderedCollections", "RigidBodyDynamics"]
-git-tree-sha1 = "b766636d31175340e542c833cff1895c2f3f41cc"
+git-tree-sha1 = "c6ca2052dc571cfe1395a12e970101c9356cf85f"
+repo-rev = "hf/upstream-changes"
+repo-url = "https://github.com/ferrolho/MeshCatMechanisms.jl.git"
 uuid = "6ad125db-dd91-5488-b820-c1df6aab299d"
-version = "0.9.0"
+version = "0.9.1"
```

Comment:
The _Manifest.toml_ files of the notebooks need to be generated again after https://github.com/JuliaRobotics/MeshCatMechanisms.jl/pull/71 has been merged. Right now, they are referring to the PR branch from my fork.